src/types/short-links-api.ts

```diff
@@ -65,5 +65,11 @@ export interface ShortLinkRequestBody {
 
 export interface ShortLinkResponse {
   shortLink: string;
+  warning?: ShortLinkWarning[];
```

Comment:
@zerefdev this field doesn't exist in their [API doc](https://firebase.google.com/docs/reference/dynamic-links/link-shortener#response_body)

Reply:
I reported that already. 
If your app is missing some configuration or any other issue, "warning" field will contain information about it.
https://firebase.google.com/docs/dynamic-links/debug#common_errors_and_warnings